src/components/visualizations/orbitalViewer/orbitalUtilities.js

```diff
@@ -3,15 +3,51 @@ import * as THREE from 'three';
 export const AU_TO_VIZ_SCALER = 100;
 export const DAY_PER_VIZ_SEC = 1;
 
+export const ORBITAL_COLORS = {
```

Comment:
created a config object that stores the object and orbital colors for each orbital object